tools/packaging/windows/aws-otel-collector.wxs

```diff
@@ -20,6 +22,12 @@ SummaryCodepage='1252'
 InstallScope="perMachine"
 Platform="x64"
     />
+
+    <MajorUpgrade
+    AllowDowngrades="no" 
+    DowngradeErrorMessage="A later version of ADOT Collector is already installed. Setup will now exit"
+    AllowSameVersionUpgrades="no" />
```

Comment:
Are these default values other than the `DowngradeErrorMessage`?

Reply:
Both the remaining parameters defaults to no